File: Pratica 1 - Calculadora Cientifica/python_logic/ROI.py
lucro = float(input("Digite o lucro: "))
investimento = float(input("Digite o investimento: "))

if investimento == 0:
    print("Erro: investimento não pode ser zero")
else:
    roi = ((lucro - investimento) / investimento) * 100
    print("ROI:", f"{roi:.2f}%")

File: Pratica 1 - Calculadora Cientifica/cpp_logic/ROI.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    double lucro, investimento;

    cout << "--- Calculadora de ROI (C++) ---" << endl;

    cout << "Digite o lucro: ";
    cin >> lucro;

    cout << "Digite o investimento: ";
    cin >> investimento;

    if (investimento == 0) {
        cout << "Erro: investimento nao pode ser zero!" << endl;
    } else {
        double roi = ((lucro - investimento) / investimento)*100;
        cout << "ROI: " << roi << "%" << endl;
    }

    return 0;
}